WordPress/Classes/Services/PostCoordinator.swift

```diff
@@ -617,7 +617,9 @@ class PostCoordinator: NSObject {
         }
     }
 
-    private func shouldScheduleRetry(for error: Error) -> Bool {
+    /// Returns `true` if the error can't be resolved by simply retrying and
+    /// requires user interventions, for example, resolving a conflict.
+    static func isTerminalError(_ error: Error) -> Bool {
```

Comment:
I think the conditions are flipped - should it be

```
    static func isTerminalError(_ error: Error) -> Bool {
        if let saveError = error as? PostRepository.PostSaveError {
            switch saveError {
            case .deleted:
                return true
            case .conflict:
                return true
            }
        }
        if let error = error as? SavingError, case .mediaFailure(_, let error) = error {
            return MediaCoordinator.isTerminalError(error)
        }
        return false
    }
```

Reply:
Oh, good catch. Fixed.